### PR TITLE
clean endpoint controller typo logs

### PR DESCRIPTION
**Pull request by yanggangtony**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
This pr is for fix endpoint controller typo logs ```pkg/controller/util/endpoint/controller_utils.go```.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: pkg/scheduler/internal/queue/scheduling_queue.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ type PreEnqueueCheck func(pod *v1.Pod) bool
 type SchedulingQueue interface {
 	framework.PodNominator
 	Add(pod *v1.Pod) error
-	// Activate moves the given pods to activeQ iff they're in unschedulablePods or backoffQ.
```
[the remark this answers is not in the document]
Comment (sanposhiho) [reply pkg/scheduler/internal/queue/scheduling_queue.go]: 👍 
(Actually, I have the experience to have sent the completely same PR https://github.com/kubernetes/kubernetes/pull/104542 😅  )

[the remark this answers is not in the document]
Comment (yanggangtony) [reply pkg/scheduler/internal/queue/scheduling_queue.go]: > 👍 (Actually, I have the experience to have sent the completely same PR #104542 😅 )

hhha

**Comment by sanposhiho** [comment]
I think there is no place on sig-scheduling side then.

/remove-sig scheduling

**Comment by danwinship** [comment]
The PR description / commit message is not really accurate since this commit touches multiple files.

You may have better luck submitting multiple PRs, so each one can be approved by an appropriate approver for the code it modifies

**Comment by yanggangtony** [comment]
@danwinship 
Thanks for your review.
And thanks to all other members's review.
I update the pr and description.


Next time , i will pay attention to this..
Thanks a lot .

**Comment by danwinship** [comment]
lgtm but can you change the commit message? Maybe "fix a reference to the wrong variable name" or something like that?

**Comment by yanggangtony** [comment]
@danwinship 
Thanks the advice.
Rebased.

**Pull request merged**